flume-ng-sources/flume-taildir-source/src/main/java/org/apache/flume/source/taildir/TaildirSource.java

```diff
@@ -104,16 +107,15 @@ public synchronized void start() {
     } catch (IOException e) {
       throw new FlumeException("Error instantiating ReliableTaildirEventReader", e);
     }
+    for (Entry<Long, TailFile> entry : reader.getTailFiles().entrySet()) {
+      inodePositionMap.put(entry.getKey() + "", entry.getValue().getPos() + "");
```

Comment:
Are the `+ ""`-s needed? As per https://stackoverflow.com/a/1572724/5323166 , `String.valueOf` is recommended. Please, let me know if the intent was different.

Reply:
Thanks for pointing out. I'll fix it.